lib/utils/validation_utils.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+
+class ValidationUtils {
+
+  static bool validateEmail(String emailValue) {
+    final emailRegex =
```

Comment:
move regex pattern constants out of the method and use that constant inside method